From b33c8b56abcc4837f96f7f106b108681858482e0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jonas=20=C3=85dahl?= <jadahl@gmail.com>
Date: Tue, 24 Apr 2018 09:46:41 +0200
Subject: [PATCH] gallium: Disable rgb10 configs by default

Applications tend to not handle rgb10 configs very well, so lets
disable it for now.

https://bugzilla.redhat.com/show_bug.cgi?id=1560481
---
 src/gallium/auxiliary/pipe-loader/driinfo_gallium.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gallium/auxiliary/pipe-loader/driinfo_gallium.h b/src/gallium/auxiliary/pipe-loader/driinfo_gallium.h
index 21dc599dc2..23f9194149 100644
--- a/src/gallium/auxiliary/pipe-loader/driinfo_gallium.h
+++ b/src/gallium/auxiliary/pipe-loader/driinfo_gallium.h
@@ -39,6 +39,6 @@ DRI_CONF_SECTION_MISCELLANEOUS
    DRI_CONF_ALWAYS_HAVE_DEPTH_BUFFER("false")
    DRI_CONF_GLSL_ZERO_INIT("false")
    DRI_CONF_VS_POSITION_ALWAYS_INVARIANT("false")
-   DRI_CONF_ALLOW_RGB10_CONFIGS("true")
+   DRI_CONF_ALLOW_RGB10_CONFIGS("false")
    DRI_CONF_ALLOW_FP16_CONFIGS("false")
 DRI_CONF_SECTION_END
